app/kafka/manager/actor/cluster/KafkaStateActor.scala

```diff
@@ -284,7 +625,8 @@ case class OffsetCacheActive(curator: CuratorFramework,
 
   protected def getTopicDescription(topic: String, interactive: Boolean) : Option[TopicDescription] = topicDescriptions(topic, interactive)
   
-  def start():  Unit = {
+  override def start():  Unit = {
+    super.start()
```

Comment:
right, sorry wrong diction :) I was thinking of not starting it if consumer polling is not enabled. Honestly, I am not sure how much of an impact this is going to have with a cluster with lots of topics. But since it shouldn't affect other actors (when consumer offsets feature is disabled), I guess leaving it as-is could be fine.